gotestcov

```diff
@@ -46,7 +46,9 @@ go test -cover -covermode=set $@ -shuffle=on -args -test.gocoverdir="${raw_cov_d
 go tool covdata textfmt -i="${raw_cov_dir}" -o="${cov_dir}/coverage.out"
 
 # Append the Rust coverage data to the Go one
-cat "${raw_cov_dir}/rust-cov/rust2go_coverage" >>"${cov_dir}/coverage.out"
+if [ -f "${raw_cov_dir}/rust-cov/rust2go_coverage" ]; then
+    cat "${raw_cov_dir}/rust-cov/rust2go_coverage" >>"${cov_dir}/coverage.out"
+fi
```

Comment:
looks good however, when there is a simgle instruction, in shell, it's usually simplified to

```
[ -f  "${raw_cov_dir}/rust-cov/rust2go_coverage" ] && cat "${raw_cov_dir}/rust-cov/rust2go_coverage" >>"${cov_dir}/coverage.out"
```

Reply:
You're way of doing it is fine too